rd_ui/app/scripts/directives/directives.js

```diff
@@ -58,7 +58,7 @@
         }
 
         $scope.$on('$locationChangeStart', function (event, next, current) {
-          if (next.split("#")[0] == current.split("#")[0]) {
+          if (next.split("?")[0] == current.split("?")[0]) {
```

Comment:
Thank you for your review. fixed https://github.com/getredash/redash/pull/1232/commits/7843d2ee84a91a316a5d048851d9bb75ead216f5